application/data_access/entity_queries.py

```diff
@@ -61,7 +61,9 @@ def get_entities(session, dataset: str, limit: int) -> List[EntityModel]:
     return [entity_factory(e) for e in entities]
 
 
-def get_entity_search(session: Session, parameters: dict):
+def get_entity_search(
+    session: Session, parameters: dict, exclude_field: Optional[List[str]] = None
```

Comment:
the exclude_field parameter should be passed in through the parameters dictionary (that's how all the other params are passed through)